File: base/Sources/strings.ts

function strings_error0(): string {
	return tr("Error: .arm file expected");
}

function strings_error1(): string {
	return tr("Error: Unknown asset format");
}

function strings_error2(): string {
	return tr("Error: Could not locate texture");
}

function strings_error3(): string {
	return tr("Error: Failed to read mesh data");
}

function strings_error5(): string {
	return tr("Error: Check internet connection to access the cloud");
}

function strings_info0(): string {
	return tr("Info: Asset already imported");
}

function strings_graphics_api(): string {
	///if krom_direct3d11
	return "Direct3D11";
	///elseif krom_direct3d12
	return "Direct3D12";
	///elseif krom_metal
	return "Metal";
	///elseif krom_vulkan
	return "Vulkan";
	///else
	return "OpenGL";
	///end
}

File: paint/sources/strings.c

#include "global.h"

char *strings_arm_file_expected() {
	return tr("Error: .arm file expected");
}

char *strings_unknown_asset_format() {
	return tr("Error: Unknown asset format");
}

char *strings_could_not_locate_texture() {
	return tr("Error: Could not locate texture");
}

char *strings_failed_to_read_mesh_data() {
	return tr("Error: Failed to read mesh data");
}

char *strings_check_internet_connection() {
	return tr("Error: Check internet connection to access the cloud");
}

char *strings_asset_already_imported() {
	return tr("Info: Asset already imported");
}

char *strings_graphics_api() {
	#ifdef IRON_DIRECT3D12
	return "Direct3D12";
	#elif defined(IRON_METAL)
	return "Metal";
	#elif defined(IRON_VULKAN)
	return "Vulkan";
	#else
	return "WebGPU";
	#endif
}
